src/poetry/utils/_compat.py

```diff
@@ -7,7 +7,7 @@
 
 if sys.version_info < (3, 8):
     # compatibility for python <3.8
-    import importlib_metadata as metadata
+    import importlib_metadata as metadata  # nopycln: import
```

Comment:
`# nopycln: import` can be replaced with `# noqa` if you prefer that.

Reply:
I'm good with this currently, since `# noqa` would cause [`python-check-blanket-noqa`](https://github.com/python-poetry/poetry/blob/16cb29cb572604f342b8157b5703aacc3ed6d25e/.pre-commit-config.yaml#L27) to complain